docs/samples/drift-detection/alibi-detect/cifar10/cifar10.yaml

```diff
@@ -1,11 +0,0 @@
-apiVersion: "serving.kubeflow.org/v1alpha2"
-kind: "InferenceService"
-metadata:
-  name: "tfserving-cifar10"
-spec:
-  default:
-    predictor:
-      tensorflow:
-        storageUri: "gs://kfserving-samples/tfserving/cifar10/resnet32"
-      logger:
-        mode: all
```

Comment:
I think these examples are still useful for yaml users like me :)

Reply:
All the yaml is in the notebook now.